Libraries/LibWeb/WebGL/WebGL2RenderingContextImpl.cpp

```diff
@@ -248,177 +248,333 @@ void WebGL2RenderingContextImpl::tex_sub_image3d(WebIDL::UnsignedLong target, We
 void WebGL2RenderingContextImpl::uniform1ui(GC::Root<WebGLUniformLocation> location, WebIDL::UnsignedLong v0)
 {
     m_context->make_current();
-    glUniform1ui(location ? location->handle() : 0, v0);
+
+    // "If the passed location is not null and was not obtained from the currently used program via an earlier call to
+    //  getUniformLocation, an INVALID_OPERATION error will be generated. If the passed location is null, the data
+    //  passed in will be silently ignored and no uniform variables will be changed."
```

Comment:
Maybe we should make this note just once for all `uniform*` methods? Feels like a lot of copy to update if the spec ever changes, and this behavior does not feel that unintuitive that it warrants so many reminders imo.